### PR TITLE
Add count parameter to rules history endpoint.

### PR DESCRIPTION
**Pull request by noddi**
Second part of feature from #1266 (second comment).

### CODE REVIEW

File: server/src/main/java/io/druid/server/http/RulesResource.java [hunk 1]
```diff
@@ -116,37 +116,52 @@ public Response getDatasourceRuleHistory(
...
-    }
+  }
+
+  private List<AuditEntry> getRuleHistory(
+      final String dataSourceName,
+      final String interval,
+      final Integer count
+  ) throws IllegalArgumentException
+  {
+      Interval theInterval = interval == null ? null : new Interval(interval);
```
Comment (himanshug) [inline server/src/main/java/io/druid/server/http/RulesResource.java]: can you move this line after the if statement below?

**Comment by nishantmonu51** [comment]
:+1:

File: server/src/main/java/io/druid/server/http/RulesResource.java (hunk 1 above)
Comment (noddi) [inline server/src/main/java/io/druid/server/http/RulesResource.java]: Sure.

**Comment by noddi** [comment]
I've changed docs too.

**Comment by himanshug** [comment]
:+1: pls squash your commits.

**Comment by noddi** [comment]
Done.

**Comment by noddi** [comment]
Travis

**Pull request merged**